101-app-configuration/azuredeploy.parameters.json

```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "adminPassword": {
+        "value": "GEN-PASSWORD"
+      },
+      "appConfigStoreName": {
```

Comment:
you need to output this before you can "GET" it... you also need the resourceGroup name that contains the store